doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -150,3 +150,21 @@ Read Client functionality
 The Read Client functionality is used for reading attributes from another device in the Matter network.
 This functionality is disabled by default for Matter samples in the |NCS|, except for ones that need to read attributes from the bound devices, such as the :ref:`matter_light_switch_sample` and :ref:`matter_thermostat_sample` samples, and the :ref:`matter_bridge_app` application.
 Enable the feature if your device needs to be able to access attributes from a different device within the Matter network using, for example, bindings.
+
+.. _ug_matter_configuration_diagnostic_logs:
+
+Diagnostic Logs
+===============
+
+The diagnostic Logs module implements all needed functionalities to allow the Matter controller to obtain the logs from the Matter devices according to the chosen intent.
+
+To use the diagnostic Logs module, add The `DiagnosticLogs` cluster in the ZAP configuration.
+To learn how to add a new cluster to the ZAP configuration see :ref:`ug_matter_gs_adding_cluster`.
+
+To enable diagnostic logs support, set :kconfig:option:`CONFIG_NCS_SAMPLE_MATTER_DIAGNOSTIC_LOGS` Kconfig option to ``y``.
+
+Currently, the following intents are defined within the ``IntentEnum`` in the Matter stack:
```

Comment:
```suggestion
Currently, the following intents are defined within the ``IntentEnum`` enumerator in the Matter stack:
```
Not sure "enumerator" is the correct type, feel free to correct.

Reply:
Yeah, I think "enumerator" is correct here :)